Orange/data/table.py

```diff
@@ -1453,6 +1453,8 @@ def transpose(cls, table, feature_names_column="",
             feature names are mapped
         :return: Table - transposed table
         """
+        from Orange.data.io import guess_data_type, sanitize_variable
```

Comment:
This import does not seem particularly slow or special in a way that would warrant local importing inside a function instead of at the top of the file. Or am I missing something?